ocs_ci/framework/conf/default_config.yaml

```diff
@@ -212,6 +212,7 @@ ENV_DATA:
   size: '20'
   ms_env_type: 'staging'
   rosa_cli_version: '1.2.2'
+  machine_cidr : "10.0.0.0/16"
```

Comment:
Do we need to have default `machine_cidr` configured here?

Reply:
Removing from here